pkgs/development/python-modules/sphinx-autobuild/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenv
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, livereload
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-autobuild";
+  version = "2020.9.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1vbaf4vrxahylyp8zrlw5dx1d2faajp926c3pl5i1wlkp1yll62b";
+  };
+
+  # No tests included.
+  doCheck = false;
+
```

```suggestion
  pythonImportsCheck = [ "sphinx_autobuild" ];
```

see https://nixos.org/manual/nixpkgs/unstable/#python

---

Added in 4aaae20